tests/test.c

```diff
@@ -429,9 +429,9 @@ void add_unordered_string_output(char* output) {
 // ************************* GAMEPAD CHECK FUNCTIONS ************************ //
 
 // Helper function to print a 32-bitmap to stderr on a new line
-static void print_bitmap(uint32_t bitmap) {
+static void print_bitmap(uint64_t bitmap) {
     uint8_t bit;
-    for (int i = 0; i < 32; i++) {
+    for (int i = 0; i < 64; i++) {
```

Comment:
Are all bitmaps that we print 64 bits? If so then this is OK. If not, I think we probably want like a second argument to this function that specifies how many bits to print, since this would fail trying to print a 32-bit int right now

Reply:
yes, its only used to print buttons.